sqlalchemy_utils/types/email.py

```diff
@@ -4,9 +4,12 @@
 
 
 class EmailType(sa.types.TypeDecorator):
-    impl = sa.Unicode(255)
+    impl = sa.Unicode
     comparator_factory = CaseInsensitiveComparator
 
+    def __init__(self, length=255):
```

Comment:
I was also thinking about that, but then I just couldn't find any compelling reason to have those options there. On the other hand I don't see any negative sides in passing those extra variables to the parent class, so I may as well do that.

I'll update this PR tomorrow.